src/external/dftk_to_wannier90.jl

```diff
@@ -0,0 +1,415 @@
+using ProgressMeter
+using LinearAlgebra
+using Dates
+using Printf
+
+## TODO
+# add handling of the SCDM routine as an alternative guess
+
+include("win_guess_utils.jl")
+
+"""
+    Create a .win file for Wannier90, compatible with the system studied with DFTK.
+    Options have to be compeleted by hand.
+    Parameters can also be added as kwargs argument : e.g. num_iter = 500.
+"""
+function dftk2wan_win_file(prefix::String, basis::PlaneWaveBasis, scfres,
+                           num_wann::Integer;
+                           bands_plot = false,
+                           kwargs...
+                           )
+    
+    # Write in file   
+    open("$prefix.win", "w") do f
+
+        write(f,"! $prefix.win file generated by DFTK at $(now())"*"\n"^2)
+        
+        # Required parameters
+        num_bands = size(scfres.ψ[1],2)
+        write(f,@sprintf("%-20s %s %5s \n","num_bands","=",num_bands))
```

Comment:
do you really have to be this careful with formating? Can't you just do `write(f, "num_bands=$num_bands")`?